index.html.md.erb

```diff
@@ -122,8 +122,9 @@ The following table provides version and version support information about Healt
 | Version | v2.3                                  |
 | Release date | April 10, 2024                        |
 | Compatible Ops Manager versions | v2.10, v3.0                             |
-| Compatible TAS for VMs versions | v2.11, v2.13, v3.0, v4.0, v5.0, v6.0          |
-| Compatible TKGI versions | v1.15, v1.16                            |
+| Compatible TAS for VMs versions | v4.0, v5.0, v6.0, v7.0          |
```

Comment:
There is no TAS v7.0
It goes up to TAS v6.0 and the new one is Tanzu Platform for Cloud Foundry v10.0